build_tools/github/build_minimal_windows_image.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+set -x
+
+PYTHON_VERSION=$1
+BITNESS=$2
+
+if [[ "$PYTHON_VERSION" == "36" || "$BITNESS" == "32" ]]; then
+    # Python 3.6 and 32-bit architectures are not
+    # yet supported by the official Docker images
+    exit 0
```

Comment:
```suggestion
    # Python 3.6 and 32-bit architectures are not yet supported by the official
    # Docker images: tests will just be run on the host instead of using a
    # minimal docker container.
    exit 0
```